### DISCUSSION TITLE
[Feature Request] 2D material

### DISCUSSION THREAD

**Discussion by JTaozhang** [deepmodeling/deepmd-kit#2934] [Ideas]
### Summary

Hi, there,

I would like to enquiry how to set the training paramters for 2D material, can I choose the same settings for 2D material as 3D material. Moreover, I notice the parameter "atom_ener" listed in the fitting_net part, whether this parameter is helpful for training 2D material? how to use this parameter?

Best regards, 

### Detailed Description

I notice the parameter "atom_ener" listed in the fitting_net part, whether this parameter is helpful for training 2D material? how to use this parameter?

**Comment by njzjz** [comment]
I need clarification on the difference in physics between 2D and 3D materials. (I think they should obey the same physical rules?)

For `atom_ener` please refer to #1357.

**Comment by JTaozhang** [comment]
Most 2D material has the in-plane periodicity and the out-of-plane direction of which is considered as vacuum. So, we can consider that the in-plane dimension of 2D is the same with 3D material, but the out-of-plane direction is totally different. The above definition is specially for monolayer, like graphene. But for bilayers or trilayers 2D material,  von der waals force and weak electron-eletron interaction always exists in the interlayer because the interlayer distance is around 3-4A. Reminding that von der waals force has a longer interaction distance than bond. So, I think if deepmd can describe the vdW force of the interlayer accurately, this software could be also used for 2D materials.

I have done some tests on 2d material using deepmd, Like change the axis_neuron, training steps, loss paramters, rcut, rcut_smooth. my conclusion is although the model predict the energy and force very accurate, the RMSE loss around(10e-3 eV/atom, 10e-3 eV/A ), the model still can't appropriately predict the structure using the force field model by lammps, especially , the out-of-plane direction, the interlayer distance always overestimate greatly.

About the traning database, The interlayer distance of the database was kept at the range of 3-4A. 

In addtion, I also have done a literature research about the usage of deepmd on 2D material. Limited by my knowledge, only a few researches (less than five) has adopted deepmd in there research. I think this is unreasonable, because deepmd has been a famous machine learning softeware in 3D material.

**Comment by njzjz** [comment]
Does this system have interaction in the range of more than the cutoff radius?

Have you tried concurrent learning?

**Comment by JTaozhang** [comment]
No,  You mean the dpgen, right? I have tried that software. But I am not familiar with that software. It seems that the labeling part highly depend on the model_devi_jobs.

**Comment by njzjz** [comment]
I'll convert the issue to discussion since there is no clear feature to request in this issue.
